tiled/server/authentication.py

```diff
@@ -1220,6 +1243,9 @@ async def revoke_session_by_id(
         return Response(status_code=HTTP_204_NO_CONTENT)
 
     async def slide_session(refresh_token, settings, db):
+        if isinstance(settings.authenticator, ProxiedOIDCAuthenticator):
+            # Session management should be handled by the client, not by Tiled.
+            return
```

Comment:
I'm not sure that passing silently is the way to go here. If we reach this codepath, something has gone wrong, right? Should this be a RuntimeError?